src/main/java/com/conversantmedia/util/estimation/Percentile.java

```diff
@@ -173,7 +173,7 @@ public float getMax() {
         return q[2*m+3];
     }
 
```

Comment:
I'm not concerned with performance here.   No need to benchmark this change, this is acceptable.